### PR TITLE
Allow JWKs to work with region code as well

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Allow JWK lookup URLs to work with region codes in addition to IDs
```

### CODE REVIEW

File: pkg/controller/jwks/jwks.go [hunk 1]
```diff
@@ -48,7 +49,7 @@ func (c *Controller) HandleIndex() http.Handler {
 		// key is the key in the cacher where the values for this JWK are cached.
 		key := &cache.Key{
 			Namespace: "jwks",
-			Key:       r.URL.String(),
+			Key:       strings.ToLower(r.URL.String()),
```
Comment (sethvargo) [inline pkg/controller/jwks/jwks.go:52]: Optimizes caching

**Review by whaught — APPROVED** [review]

**Review by mikehelmick — APPROVED** [review]

**Comment by sethvargo** [comment]
Well, I had to make it less elegant unfortunately.

**Comment by sethvargo** [comment]
@whaught PTAL

**Review by whaught — APPROVED** [review]

**Pull request merged**